core/terraform/airlock/variables.tf

```diff
@@ -62,6 +62,9 @@ variable "airlock_servicebus" {
     default_primary_connection_string = string
```

Comment:
Have you tried to add the `endpoint` property here and checking if it will get populated automatically?

Reply:
yes, it is, refactored